spec/controllers/ideas_controller_spec.rb

```diff
@@ -24,11 +24,11 @@
   # Idea. As you add validations to Idea, be sure to
   # adjust the attributes here as well.
   let(:valid_attributes) {
-    { title: "valid title", body: "valid body", quality: "0"}
+    { title: "valid title", body: "valid body"}
   }
 
   let(:invalid_attributes) {
-    {body:'', title:'', quality:''}
+    {body:'', title:''}
```

Comment:
Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.